client/src/layouts/BaseLayout.scss

```diff
@@ -2,7 +2,9 @@
 @import 'styles/helpers';
 
 body {
-  background-color: $color-primary;
+  background-image: url('../images/backgound-light-grey.png');
```

Comment:
Remove all unused images and rename the remaining one `background.png` instead of `backgound-light-grey.png`.

Reply:
I left them so we could try different layouts, but if you're ok with this version it's fine by me.